core/controllers/topics_and_skills_dashboard.py

```diff
@@ -251,7 +251,7 @@ class NewTopicHandler(base.BaseHandler):
     def post(self):
         """Handles POST requests."""
         name = self.payload.get('name')
-        abbreviated_name = self.payload.get('abbreviated_name')
+        abbreviated_name = self.payload.get('abbreviatedName')
```

Comment:
This is odd, all these should be snake_case. Please fix the frontend if it's passing the wrong things?